app/services/projectList.js

```diff
@@ -176,9 +176,9 @@ export default class ProjectListService {
     // is null and the other is a blank string, it's okay if they match.
     return projects.sort((a, b) => {
       // eslint-disable-next-line prettier/prettier
-      const stringA = (a && a.name) ? a.name.toLowerCase() : Constants.UndefinedDefaults.PROJECT;
+      const stringA = a && a.name ? a.name.toLowerCase() : Constants.UndefinedDefaults.PROJECT;
       // eslint-disable-next-line prettier/prettier
-      const stringB = (b && b.name) ? b.name.toLowerCase() : Constants.UndefinedDefaults.PROJECT;
+      const stringB = b && b.name ? b.name.toLowerCase() : Constants.UndefinedDefaults.PROJECT;
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line on 180